src/test/java/jssc/SerialNativeInterfaceTest.java

```diff
@@ -47,4 +55,16 @@ public void reportsWriteErrorsAsIOException() throws Exception {
         testTarget.writeBytes(fd, buf);
     }
 
+    @Test
+    public void throwsNpeIfPassedBufferIsNull() throws Exception {
+        long fd = 1;
+        byte[] buf = null;
+        try{
+            testTarget.writeBytes(fd, buf);
+            fail("Where is the exception?");
+        }catch( NullPointerException ex ){
+            assertTrue(ex.getMessage().contains("buffer"));
+        }
+    }
```

Comment:
Should we copy the `expected =` signature in case we change the messaging later?

```suggestion
    @Test(expected = java.lang.NullPointerException.class)
    public void throwsNpeIfPassedBufferIsNull() throws Exception {
        long fd = 1;
        byte[] buf = null;
        testTarget.writeBytes(fd, buf);
    }
```